generators/python-v2/ast/src/__test__/__snapshots__/PythonFile.test.ts.snap

```diff
@@ -112,6 +112,17 @@ my_variable: ImportedClass.nested.attribute
 "
 `;
 
+exports[`PythonFile > Write duplicative import names 1`] = `
+"from .cars import Car
+from .vehicles import Car
+
+
+class Car:
+    car = Car()
+    vehicle = Car()
```

Comment:
This seems like the harder case. Here, we create a class called `Car`, which has a naming conflict with the imports above. Do other language ast modules handle this case? Is it generally left up to the consumer to avoid? One could argue the desired generated code should look like:

```python
from .cars import Car as _Car
from .vehicles import Car as __Car

class Car:
    car = _Car()
    vehicle = __Car()
```

Reply:
Yup the AST should allow a consumer to pass in any overrides to the alias (i.e. Fern has important info that allows us to alias in a nicer way)

Reply:
> Should the AST be smart enough to see that lower down in the file, we have a class named Car, therefore it should automatically produce an alias for all imports of other objects named Car

This one - the AST should know how to guarantee that it can write itself to a file, and do so in a deterministic way.